exercises/bob/introduction.md

```diff
@@ -0,0 +1,20 @@
+# Introduction
+
+Bob is a teenager.
+He's not very smart, but he likes to think that he's very cool.
+He definitely doesn't get excited about things.
+That wouldn't be cool.
+
+When people talk to him, his responses are pretty limited.
+In fact, he only ever answers one of five things:
+
+- _"Sure."_
+  This is his response if you ask him a question, such as "How are you?"
+- _"Whoa, chill out!"_
+  This is his answer if you YELL AT HIM (in all capitals).
+- _"Calm down, I know what I'm doing!"_
+  This is what he says if you yell a question at him.
+- _"Fine. Be that way!"_
+  This is how he responds if you address him without actually saying anything.
+- _"Whatever."_
+  This is what he answers to anything else.
```

Comment:
Personally, I like this version better! It makes it more clear that there are exactly five response and their definitions are now grouped together, which I like.

Reply:
I have a preference for bold (`**`) not italic (`_`) formatting as I feel it makes it stand out more on the website. I think italics things tend to blur into the background more, so despite `_` being for emphasis, I actually think it has the opposite UX effect when rendered. So I'd change these to bold.